### ISSUE TITLE
is it working with new iOS and Xcode 9?

### ISSUE THREAD

**Issue by runtoapp** [Mahabali/BonjourWebrtc#3]
I tested this source with iPhone 5 (10.3.3) and iPhone 6s(11.3.X)...

before that I upgrade your source to Swift 4.

and I tested it but always the logs shows "ICEStopConnectivityCheck() found no ICE check with call id (XXX), Browser got invite response with nil connectionData., Not in connected state, so giving up for participant [XXXXXXX] on channel [0]."

Thanks for your wonderful works and May I use your source??

**Comment by Mahabali** [comment]
Yes you can use without any conditions. Its been a long time since i updated webrtc part. I may update it in future. As of now i am busy with other works !
